### ISSUE TITLE
Prepare and submit NBIB test case # 7

### ISSUE THREAD

**Issue by michaelccata** [18F/e-QIP-prototype#744]
Waiting on Ed at the moment. Full blown case similar to https://github.com/18F/e-QIP-prototype/issues/725

Check for phantom validation errors and open separate ticket as required.

**Comment by ryanhofdotgov** [comment]
Initial version of test case is prepped, but about ~80 zip code errors. Waiting until bulk of issues from https://github.com/18F/e-QIP-prototype/issues/743 are addressed before cleaning up/submitting.

**Comment by ryanhofdotgov** [comment]
eApp UI issues that I've worked around so far by manually editing the test data in order to get test 7 valid and accepted by e-QIP:
https://github.com/18F/e-QIP-prototype/issues/860
https://github.com/18F/e-QIP-prototype/issues/859
https://github.com/18F/e-QIP-prototype/issues/863
https://github.com/18F/e-QIP-prototype/issues/899

These issues, once resolved would have prevented a user from creating the invalid data.
